Mathlib/LinearAlgebra/Prod.lean

```diff
@@ -864,6 +864,25 @@ theorem skewProd_symm_apply (f : M →ₗ[R] M₄) (x) :
 
 end
 
+section Unique
+
+variable {N : Type w}
+variable [Semiring R]
+variable [AddCommMonoid M] [AddCommMonoid N]
+variable [Module R M] [Module R N]
+
+/-- Multiplying by the trivial module from the left does not change the structure.-/
```

Comment:
```suggestion
/-- Multiplying by the trivial module from the left does not change the structure.

This is the linear version of `AddEquiv.uniqueProd`. -/
```
And same throughout, since this creates links in the web documentation.